themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1143,6 +1147,32 @@ button.tc-btn-invisible.tc-remove-tag-button {
 	}
 }
 
+
+/* prevent overflow for table content for small screens */
+@media (max-width: <<tiddler-width>>) {
+	tr {
```

Comment:
This definition globally redefines the `<tr>` element, regardless of whether it is used inside a tiddler. So that means that (say) adding a table to a new menubar would be hit by this rule.

Reply:
Yes that's intentional for tables with small screens.

Reply:
On small screens it also half's the indent distance for all list types, to improve the readability for "real" text instead of showing a lot of whitespace. See the animated gif

Reply:
The trouble is that these definitions are very broad. They would hit content embedded in widgets by third party libraries, for example. I do appreciate that that is also true of our table CSS generally, but there's no reason to make that worse.